README.md

```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
     --s3.endpoint="$S3_ENDPOINT"
 ```
 
+## Tiup Quick Start
+
+```sh
+# Using tiup to start a TiDB cluster.
+tiup playgroup
+
+# Using tiup bench to generater test data.
+tiup bench tpcc --warehouses 1 prepare
+
+# How many row do we get? 300315 rows.
+mysql --host 127.0.0.1 --port 4000 -E -e "SELECT COUNT(*) FROM test.customer" -u root -p
```

Comment:
If we selected from `test.customer` before backing up, then select from `test.order_line` after restoration, we would probably get unexpected result.

Reply:
> If we selected from `test.customer` before backing up, then select from `test.order_line` after restoration, we would probably get unexpected result.

I'm sorry, this is a typo, I should check more carefully. `test.customer` should be corrected to `test.order_line`.

Reply:
Never mind, typo is a component of life...

Reply:
> Never mind, typo is a component of life...

I will redo a backup && restore test quickly and correct my pr.